### PR TITLE
 Fix scoresheet, scoresheets, and scoresheet entries

### PR DESCRIPTION
**Pull request by Bhammy**
Scoresheets.jsx now correctly updates on a receiveScoresheet action; this cascades down so that score entries correctly update.

### CODE REVIEW

**Comment by Bhammy** [comment]
Newest branch calls fetchContests on componentDidMount for new scoresheet form so as to fill dropdown.

File: src/components/scoresheets/new_scoresheet.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  handleChange(field) {
+    return (e) => {
+      let newForm = this.state.scoresheet;
+      newForm[field] = e.currentTarget.value;
+      this.setState({ scoresheet: newForm })
+    }
+  }
+
+  submitForm() {
+    debugger
```
Comment (matthaws) [inline src/components/scoresheets/new_scoresheet.jsx]: debugger?

File: src/components/scoresheets/scoresheet_entry.jsx [hunk 2]
```diff
@@ -1,192 +1,205 @@
...
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
-import { createScoring, updateScoring } from '../../actions/scoringActions';
+import { createScoring, updateScoring } from "../../actions/scoringActions";
 import YouTube from "../video/YouTube";
 import "./scoresheet_entry.css";
 
 //=========================================
-// mapDispatchToProps
+// mapXToProps
```
Comment (matthaws) [inline src/components/scoresheets/scoresheet_entry.jsx:9]: ?

File: src/components/scoresheets/scoresheet_entry.jsx [hunk 3]
```diff
@@ -1,192 +1,205 @@
...
 
-const mapDispatchToProps = (dispatch) => ({
-  createScoring: scoring => dispatch(createScoring(scoring)),
-  updateScoring: scoring => dispatch(updateScoring(scoring)),
+const mapStateToProps = (state, ownProps) => ({
```
Comment (matthaws) [inline src/components/scoresheets/scoresheet_entry.jsx]: if mapStateToProps is not being used, let's remove it and pass null in as the first argument to connect

**Review by matthaws — CHANGES_REQUESTED** [review]
Remove debugger and extraneous mapStateToProps

**Review by matthaws — APPROVED** [review]

**Pull request merged**